docs/querying/sql.md

```diff
@@ -409,3 +409,26 @@ To solve this issue, explicitly provide the type of the dynamic parameter using
 ```
 SELECT * FROM druid.foo WHERE dim1 like CONCAT('%', CAST (? AS VARCHAR), '%')
 ```
+
+Dynamic parameters can also replace arrays, reducing the parsing time.
+
+for example:
+```json
+{
+    "query": "SELECT doubleArrayColumn from druid.table where ARRAY_CONTAINS(?, doubleArrayColumn)",
+    "parameters": [
+      {"type":"ARRAY", "value":[-25.7, null, 36.85]}
+    ]
+}
+```
```

Comment:
I feel like this is `sql-api` specific so it shouldn't it be inside `../api-reference/sql-api.md` ?

Reply:
made these simple sql queries and provided the example parameters inside `sql-api.md`